stable/database/README.md

```diff
@@ -1,33 +1,18 @@
 # NuoDB Database Helm Chart
```

Comment:
this document does not contain the "how to delete this deployment" section, that some others do

Reply:
This chart doc page for database includes at the end as many of the other pages do.

> Uninstalling the Chart
> To uninstall/delete the deployment:
> 
> helm del --purge database
> The command removes all the Kubernetes components associated with the chart and deletes the release.